frontend/src/pages/WorkspaceTeam/WorkspaceTeam.tsx

```diff
@@ -55,18 +55,9 @@ const WorkspaceTeam = () => {
 		update(cache, { data }) {
 			cache.modify({
 				fields: {
-					workspace_admins(existingAdmins, { readField }) {
-						if (data?.deleteAdminFromWorkspace !== undefined) {
-							message.success('Removed member')
-							return existingAdmins.filter(
-								(admin: any) =>
-									data.deleteAdminFromWorkspace !==
-									readField('id', admin),
-							)
-						}
-						message.success('Failed to remove member')
-						return existingAdmins
-					},
+					workspace_admins(_existingAdmins, { DELETE }) {
+						return DELETE
+					}
 				},
 			})
 		},
```

Comment:
nice @d29107d ! would this update even if the backend returns an error?
can also add
```
refetchQueries: [namedOperations.Query.GetWorkspaceAdmins],
```
to make sure the operation succeeds server side before updating the UI

Reply:
> would this update even if the backend returns an error?
I am not sure.
I add the old condition to make sure catch the error.